pkg/kubelet/cm/cpuset/cpuset.go

```diff
@@ -172,6 +172,16 @@ func (s CPUSet) ToSlice() []int {
 	return result
 }
 
+// ToSliceNoSort returns a slice of integers that contains all elements from
```

Comment:
Nit: I think that the sorted slice method should use the unsorted slice method for constructing the actual slice, and then just call sort on the result.

Reply:
I don't have strong opinion.

Let's wait for the next reviewer's comment.